test/data/test_functional.py

```diff
@@ -74,6 +80,96 @@ def test_sentencepiece_tokenizer(self):
+    #     test_sample = '\'".<br />,()!?;:   Basic English Normalization for a Line of Text   \'".<br />,()!?;:'
+    #     ref_results = ["'", '.', ',', '(', ')', '!', '?', 'basic', 'english', 'normalization',
+    #                    'for', 'a', 'line', 'of', 'text', "'", '.', ',', '(', ')', '!', '?']
+
+    #     basic_english_normalize = BasicEnglishNormalize()
+    #     experimental_eager_tokens = basic_english_normalize(test_sample)
+
+    #     jit_basic_english_normalize = torch.jit.script(basic_english_normalize)
+    #     experimental_jit_tokens = jit_basic_english_normalize(test_sample)
+
+    #     basic_english_tokenizer = data.get_tokenizer("basic_english")
+    #     eager_tokens = basic_english_tokenizer(test_sample)
+
+    #     self.assertEqual(experimental_jit_tokens, ref_results)
+    #     self.assertEqual(experimental_jit_tokens, eager_tokens)
+    #     self.assertEqual(experimental_jit_tokens, experimental_eager_tokens)
+
+    def test_BasicEnglishNormalize(self):
+        test_sample = 'Basic English Normalization for a Line of Text'
+        ref_results = ['basic', 'english', 'normalization',
+                       'for', 'a', 'line', 'of', 'text']
+
+        basic_english_normalize = BasicEnglishNormalize()
+        experimental_eager_tokens = basic_english_normalize(test_sample)
+
+        basic_english_tokenizer = data.get_tokenizer("basic_english")
+        tokens_eager = basic_english_tokenizer(test_sample)
+
+        self.assertEqual(experimental_eager_tokens, ref_results)
+        self.assertEqual(experimental_eager_tokens, tokens_eager)
+
+    # TODO(Nayef211): uncomment and replace the test below with this once once
+    # https://github.com/pytorch/pytorch/issues/38207 is closed
+    # def test_RegexTokenizer(self):
+    #     test_sample = '\'".<br />,()!?;:   Basic Regex Tokenization for a Line of Text   \'".<br />,()!?;:'
+    #     ref_results = ["'", '.', ',', '(', ')', '!', '?', 'Basic', 'Regex', 'Tokenization',
+    #                    'for', 'a', 'Line', 'of', 'Text', "'", '.', ',', '(', ')', '!', '?']
+    #     patterns_list = [
+    #         (r'\'', ' \'  '),
+    #         (r'\"', ''),
+    #         (r'\.', ' . '),
+    #         (r'<br \/>', ' '),
+    #         (r',', ' , '),
+    #         (r'\(', ' ( '),
+    #         (r'\)', ' ) '),
+    #         (r'\!', ' ! '),
+    #         (r'\?', ' ? '),
+    #         (r'\;', ' '),
+    #         (r'\:', ' '),
+    #         (r'\s+', ' ')]
+
+    #     regex_tokenizer = RegexTokenizer(patterns_list)
+    #     eager_tokens = regex_tokenizer(test_sample)
+
+    #     jit_regex_tokenizer = torch.jit.script(regex_tokenizer)
+    #     jit_tokens = jit_regex_tokenizer(test_sample)
+
+    #     self.assertEqual(jit_tokens, ref_results)
+    #     self.assertEqual(jit_tokens, eager_tokens)
+
+    def test_RegexTokenizer(self):
+        test_sample = 'Basic Regex Tokenization for a Line of Text'
+        ref_results = ['Basic', 'Regex', 'Tokenization',
```

Comment:
This isn't really engaging any of the patterns in the list

Reply:
you're right. let me update this test